docs/theming/developers.md

```diff
@@ -44,13 +44,108 @@ and elements. In other words, theming is the developer’s process of
 orchestrating design tokens with elements, particularly by way of themeable 
 container elements.
 
+## How theming works
+
+### Color Scheme Providers
```

Comment:
As a heads up, our headings and page titles should be using sentence case. I think this is something we'll have to check for across UX dot, but I thought they could be fixed for the Theming section while it's being worked on anyway.

Reply:
fixed